packages/nx-openapi/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aligent/nx-openapi",
-  "version": "1.0.0",
```

Comment:
This change is intended. We have never ever release this before. It should stay in beta for us to test out.